api/python/cellxgene_census/tests/experimental/pp/test_online.py

```diff
@@ -37,7 +38,10 @@ def test_meanvar(matrix: sparse.coo_matrix, n_batches: int, stride: int, ddof: i
     assert n_samples.sum() == matrix.shape[0]
     assert len(n_samples) == n_batches
 
-    olmv = MeanVarianceAccumulator(n_batches, n_samples, matrix.shape[1], ddof)
+    # exclude_zeros only if there is a single batch
```

Comment:
missing test to confirm that the code raises if `batches>1 and exclude_zeros`

Reply:
Added as a separate test.